projects/arcus.cmake

```diff
@@ -49,6 +56,8 @@ if(BUILD_OS_WINDOWS)
         CMAKE_ARGS -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
                    -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
                    -DCMAKE_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib-mingw
+                   -DCMAKE_CXX_STANDARD=17
+                   -DCMAKE_CXX_FLAGS="-std=c++17"
```

Comment:
Seems kind of double? `CMAKE_CXX_STANDARD` should already enforce the C++17 standard, right?

Reply:
> You're using custom C++ flags in several places, which I think should be implemented with the built-in CMake variable which is expected to set the correct flags on every platform with whatever compiler is being used.

When I tried, it seemed that the CXX_STANDARD one alone wasn't enough unfortunately, at least not on Windows iirc.

Reply:
Ignore my previous message, please: Just saw what I did for the engine ... which is _not_ with the extra cxx flags. but only with the standard.  I'll fix all of these.